### DISCUSSION TITLE
Making such charts possible?

### DISCUSSION THREAD

**Discussion by anime-shed** [illacloud/illa-builder#1965] [Q&A]
I have some sample charts that I wanted to create from the ill-builder, but I need help finding ways to modify them to achieve what I wanted, so I will be glad if it is possible.

![image](https://github.com/illacloud/illa-builder/assets/67512697/adf5c7aa-9ffb-44cb-b678-275a339b837a)
![image](https://github.com/illacloud/illa-builder/assets/67512697/aedd1666-9b12-4cdc-a36f-9f903bc9f1a6)
![image](https://github.com/illacloud/illa-builder/assets/67512697/c51e28af-5490-44ea-9ba1-1a211e254209)
![image](https://github.com/illacloud/illa-builder/assets/67512697/5300b0bc-6aec-4c2f-97c2-622e2d84a4e9)

**Comment by wenzhuo2022** [comment]
We will launch horizontal bar charts and pie charts this week. For the last type of chart, you can try using the circle progress component.

**Reply by anime-shed** [reply]
I wanted to clarify if the horizontal bar that is being released, will allow overlay bars with a target or stacked bars with a target?
